### PR TITLE
Refactor ast builder apis

### PR DESCRIPTION
**Pull request by madou**
- Refactor `ast-builder` apis
- Renames `Cache` to `cache`

### CODE REVIEW

File: packages/babel-plugin/src/utils/ast-builders.tsx [hunk 1]
```diff
@@ -68,15 +89,30 @@ const buildCssVariablesProp = (
...
 
-const buildComponentTag = ({ name, type }: Tag) =>
-  type === 'InBuiltComponent' ? `"${name}"` : name;
+/**
+ * Returns a tag string in the form of an identifier or string literal.
+ *
+ * A type of InBuiltComponent will return a string literal,
+ * otherwise an identifier string will be returned.
+ *
+ * @param param0
```
Comment (pgmanutd) [inline packages/babel-plugin/src/utils/ast-builders.tsx:104]: nit: Should we add description for parameters here?

**Review by pgmanutd — APPROVED** [review]
Just one nit is there. Otherwise lgtm 😃

File: packages/babel-plugin/src/utils/ast-builders.tsx (hunk 1 above)
Comment (madou) [reply packages/babel-plugin/src/utils/ast-builders.tsx:104]: probably 🤔 😅

**Comment by madou** [comment]
@pgmanutd done

**Review by pgmanutd — APPROVED** [review]

**Pull request merged**